config/initializers/merit.rb

```diff
@@ -0,0 +1,34 @@
+# Use this hook to configure merit parameters
+Merit.setup do |config|
```

Comment:
Unused block argument - `config`. You can omit the argument if you don't care about it.